bin/codacy-analysis-cli.sh

```diff
@@ -27,8 +27,8 @@ test_docker_socket() {
 
 run() {
   local output_volume="";
-  if [ -n "${OUTPUT}" ]; then
-    output_volume="--volume ${OUTPUT}:${OUTPUT}";
+  if [ -n "${OUTPUT_DIRECTORY}" ]; then
+    output_volume="--volume ${OUTPUT_DIRECTORY}:${OUTPUT_DIRECTORY}";
```

Comment:
Will this work if `OUTPUT_DIRECTORY` is the same as `CODACY_CODE`?

Reply:
Yes